content/back-to-basics-automation.md

```diff
@@ -0,0 +1,65 @@
+
+> ![Trust](../images/back-to-basics-automation-1.jpg) 
+
+If trust is the salt in automation, what is the pepper?
+
+My former leader and mentor always reminds me to ensure that we can answer the question: “_what is in it for me_?” Automation provides efficiency gains, accuracy, scalability, cost reduction, faster time to market, consistency, reliability, compliance, auditability, and enables continuous improvement. However, it does not address “the” question and often evokes concerns among engineers about making humans redundant, reminiscent of the industrial revolution.
+
+We need messaging that addresses the mentioned benefits and emphasizes that automation eliminates many repetitive tasks. This allows engineers to spend more time on innovative and impactful work. For instance, generative AI, self-service automation, and upgrade automation are designed to assist engineers by freeing up time, reducing mental fatigue, and increasing creativity. The goal is not to make human roles redundant but to balance the work-life dynamic and enable improved, faster performance. 
+
+Importantly, the time saved through automation should not solely be used to increase business output, as this would undermine the advantages and progress made with automation. 
+
+> ![Salt and Pepper](../images/back-to-basics-automation-2.jpg) 
+
+So, if **trust** is the salt, knowing “**_what is in it for me_**” is the pepper!
+
+---
+
+# The **VALUE** of automation
+
+So what, apart from obvious benefits, what are some key values of automation?
+
+- Automation excels at **repetitive** tasks.
+- Automation speeds up repetitive tasks like testing, deployment, and data processing, increasing productivity.
+- Automation reduces **human error**, ensuring **consistent** and accurate task performance.
+- Automation allows users to **focus** more on complex problems, boosting job **satisfaction**, **innovation**, and productivity. I intentionally did not **bold** productivity, as some engineers may question this with "_So, I am not productive?_", which is obviously far from the intent and truth.
+
+Automation with human involvement is a win-win. Together they create a symbiotic relationship that drives true progress.
+
+---
+
+# The **DARK** side of automation
+
+Although automation has become a vital part of engineering environments, it also has a dark side we need to be aware of.
+
+- Automation lacks **creativity** and **intuition** for solving complex problems.
+- Each organization needs **customized** and **optimized automation** - a challenge evident from our shared [v2 CI/CD blueprints](https://github.com/WorkSafeBC-Common-Engineering/AzureDevOps.Automation.Pipeline.Templates.v2).
+- **Human oversight** is essential to prevent errors. Have you ever considered living next to an automated nuclear plant or flying in a cockpit-less aircraft.
```

Comment:
just needs a "?" at the end, as it is a question ;)

Reply:
Will fix in final master.